src/api/BackendConnector.ts

```diff
@@ -1,25 +1,63 @@
-import { Session } from "inspector";
+import { SessionSend, SessionGet } from "./Session";
+import { ParkerStatistic } from "./ParkerSatistic";
 
 class BackendConnector {
-    static createSession(userID: number, kennzeichen: string) {
-        const currentDate = new Date();
-        //TODO
+    static apiUrl: string = `${window.location.protocol}//${window.location.hostname}:3000`
```

Comment:
Wenn der Server auf dem gleichen System läuft, dann ist es nicht möglich, die Webseite unter `localhost` zu verwenden.

Ich denke, das sollte so klappen:
```suggestion
    static apiUrl: string = window.location.hostname === 'localhost' ? `${window.location.protocol}//127.0.0.1:3000` : `${window.location.protocol}//${window.location.hostname}:3000`
```